<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CfgEditUnit</class>
 <widget class="QDialog" name="CfgEditUnit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EditUnit</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" colspan="2">
         <widget class="QTreeWidget" name="treeWidget_listAct">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>动作列表</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>Key</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Script</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="3,0">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
         <item>
          <widget class="QTreeWidget" name="treeWidget_EditAct">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <column>
            <property name="text">
             <string>Action</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>参数</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Unit:</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0" colspan="2">
             <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,4,0,0,0">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Name:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_UnitName"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Cycle:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBox_UnitCycle"/>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Description:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTextEdit" name="textEdit_UnitDes"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_unitSave">
                <property name="text">
                 <string>Save</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_unitLook">
                <property name="text">
                 <string>Look</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDialogButtonBox" name="buttonBox">
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="standardButtons">
                 <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget_ActConfig">
         <property name="tabPosition">
          <enum>QTabWidget::North</enum>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tab_properties">
          <attribute name="title">
           <string>Properties</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="1" rowspan="3">
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="0" rowspan="3">
            <widget class="QTreeWidget" name="treeWidget_properties">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <attribute name="headerVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>属性</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>参数</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>编辑</string>
              </property>
             </column>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="pushButton_Changed">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Changed</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_DataSource">
          <attribute name="title">
           <string>Data Source</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QTabWidget" name="tabWidget_DataSource">
             <property name="tabPosition">
              <enum>QTabWidget::West</enum>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="tab_Waittime">
              <attribute name="title">
               <string>Waittime</string>
              </attribute>
              <layout class="QGridLayout" name="gridLayout_7">
               <item row="0" column="0" colspan="2">
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>Waittime(mS):</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_waittime"/>
                 </item>
                </layout>
               </item>
               <item row="0" column="2" rowspan="3">
                <widget class="Line" name="line_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                </widget>
               </item>
               <item row="0" column="3">
                <widget class="QPushButton" name="pushButton_timeAdd">
                 <property name="maximumSize">
                  <size>
                   <width>75</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>ADD</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QCheckBox" name="checkBox_timeChanged">
                 <property name="toolTip">
                  <string>随Unit循环处理；</string>
                 </property>
                 <property name="text">
                  <string>变动</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0" colspan="2">
                <widget class="QGroupBox" name="groupBox_timeChange">
                 <property name="title">
                  <string/>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_6">
                  <item row="1" column="0" colspan="2">
                   <widget class="QLabel" name="label_7">
                    <property name="text">
                     <string>Range(mS)：</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QRadioButton" name="radioButton_timeInc">
                    <property name="text">
                     <string>递增</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1" colspan="2">
                   <widget class="QRadioButton" name="radioButton_timeDec">
                    <property name="text">
                     <string>递减</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="2">
                   <widget class="QLineEdit" name="lineEdit_timeStep"/>
                  </item>
                  <item row="1" column="3">
                   <widget class="QLineEdit" name="lineEdit_timeMax"/>
                  </item>
                  <item row="2" column="0" colspan="2">
                   <widget class="QLabel" name="label_9">
                    <property name="text">
                     <string>步进(mS)：</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="2">
                   <widget class="QLineEdit" name="lineEdit_timeMin"/>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tab_Check">
              <attribute name="title">
               <string>Check</string>
              </attribute>
              <layout class="QGridLayout" name="gridLayout_9">
               <item row="2" column="1">
                <widget class="QGroupBox" name="groupBox_script">
                 <property name="title">
                  <string>Script</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <property name="checked">
                  <bool>false</bool>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_16">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_10">
                    <property name="text">
                     <string>CheckLog:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="lineEdit_scriptString">
                    <property name="text">
                     <string>OK (1 test)</string>
                    </property>
                    <property name="readOnly">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="0" column="1" rowspan="2">
                <widget class="QGroupBox" name="groupBox_volt">
                 <property name="title">
                  <string>Volt</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <property name="checked">
                  <bool>false</bool>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_14">
                  <item row="0" column="0">
                   <widget class="QComboBox" name="comboBox_vJudge">
                    <item>
                     <property name="text">
                      <string>&gt;=</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>&gt;</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>&lt;=</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>&lt;</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>==</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>!=</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>&gt;= &amp;&amp; &lt;=</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_8">
                    <property name="text">
                     <string>VoltRange:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLineEdit" name="lineEdit_voltMin"/>
                  </item>
                  <item row="1" column="2">
                   <widget class="QLineEdit" name="lineEdit_voltMax"/>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QGroupBox" name="groupBox_sound">
                 <property name="title">
                  <string>Sound</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <property name="checked">
                  <bool>false</bool>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_15">
                  <item row="0" column="0">
                   <widget class="QComboBox" name="comboBox_soundChanged">
                    <item>
                     <property name="text">
                      <string>当前有声音</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>当前无声音</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>存在有声音值</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>存在无声音值</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>有声音次数大于无声音次数</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>有声音次数小于无声音次数</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="0" column="0" rowspan="2">
                <widget class="QGroupBox" name="groupBox_cur">
                 <property name="title">
                  <string>Current</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <property name="checked">
                  <bool>false</bool>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_8">
                  <item row="0" column="0">
                   <widget class="QComboBox" name="comboBox_cJudge">
                    <item>
                     <property name="text">
                      <string>&gt;=</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>&gt;</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>&lt;=</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>&lt;</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>==</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>!=</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>&gt;= &amp;&amp; &lt;=</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_6">
                    <property name="text">
                     <string>CurrentRange:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="3">
                   <widget class="QLineEdit" name="lineEdit_curMin"/>
                  </item>
                  <item row="1" column="4">
                   <widget class="QLineEdit" name="lineEdit_curMax"/>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="0" column="2" rowspan="3" colspan="2">
                <widget class="Line" name="line_3">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                </widget>
               </item>
               <item row="0" column="4">
                <widget class="QPushButton" name="pushButton_checkAdd">
                 <property name="maximumSize">
                  <size>
                   <width>75</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>ADD</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_Module">
          <attribute name="title">
           <string>Module</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_13">
           <item row="0" column="0">
            <widget class="QCheckBox" name="checkBox_voltChanged">
             <property name="text">
              <string>电压变动</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2" rowspan="2">
            <widget class="Line" name="line_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QPushButton" name="pushButton_voltChangeAdd">
             <property name="maximumSize">
              <size>
               <width>75</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>ADD</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1" rowspan="2">
            <widget class="QGroupBox" name="groupBox_voltChange">
             <property name="title">
              <string/>
             </property>
             <layout class="QGridLayout" name="gridLayout_12">
              <item row="0" column="0">
               <widget class="QRadioButton" name="CTestCV_radioButton_a">
                <property name="text">
                 <string>递增</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="1" colspan="2">
               <widget class="QRadioButton" name="CTestCV_radioButton_b">
                <property name="text">
                 <string>递减</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0" colspan="2">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>VoltRange(V)：</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QLineEdit" name="CTestCV_min_lineEdit"/>
              </item>
              <item row="1" column="3">
               <widget class="QLineEdit" name="CTestCV_max_lineEdit"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>步进(V)：</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QLineEdit" name="CTestCV_step_lineEdit"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CfgEditUnit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>902</x>
     <y>472</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CfgEditUnit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>902</x>
     <y>472</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
